src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -12,6 +12,7 @@
 using System.Threading;
```

Comment:
@SteveL-MSFT, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

SxS Module Path Basic Tests.Windows PowerShell does not inherit PowerShell paths
```powershell
Expected like wildcard '*\powershell\*' to not match 'D:\a\1\s\test\tools\Modules;D:\a\1\s\test\tools\Modules;c:\program files\powershell\6\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\windows\system32\WindowsPowerShell\v1.0\Modules', but it did match.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Module\ModulePath.Tests.ps1: line 160
160:         $out | Should -Not -BeLike '*\powershell\*'
```
TabCompletion.Should complete abbreviated function
```powershell
Expected exactly 'PSConsoleHostReadLine', but got @('PSConsoleHostReadLine', 'PSReadLine\PSConsoleHostReadLine').
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 22
22:         $res | Should -BeExactly 'PSConsoleHostReadLine'
```
Certificate Provider tests.Error occurred in Describe block
```powershell
Object reference not set to an instance of an object.
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\certificateCommon.psm1:157 char:13
```
CmsMessage cmdlets thorough tests.Error occurred in Describe block
```powershell
Object reference not set to an instance of an object.
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\certificateCommon.psm1:157 char:13
```
Import-Module.should be able to add a module with using ModuleInfo switch
```powershell
Expected exactly 'Microsoft.PowerShell.Security', but got @('Microsoft.PowerShell.Security', 'Microsoft.PowerShell.Security').
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Import-Module.Tests.ps1: line 26
26:         (Get-Module -Name $moduleName).Name | Should -BeExactly $moduleName
```